src/main/java/com/java_essential/Cherednichenko/extra_task/CustomerService/SupportService.java

```diff
@@ -0,0 +1,206 @@
+package com.java_essential.Cherednichenko.extra_task.CustomerService;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class SupportService {
```

Comment:
This support service is unnecessary!
Use only new class
```suggestion
public class SupportService {
```